connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -339,9 +339,16 @@ private PluginScanResult scanPluginPath(
             Class<T> klass,
             ClassLoader loader
     ) throws InstantiationException, IllegalAccessException {
-        Set<Class<? extends T>> plugins = reflections.getSubTypesOf(klass);
-
         Collection<PluginDesc<T>> result = new ArrayList<>();
+        Set<Class<? extends T>> plugins;
+        try {
+             plugins = reflections.getSubTypesOf(klass);
+        } catch (ReflectionsException e) {
+            log.debug("Reflection scanner could not find any classes for URLs {}",
```

Comment:
nit: plural (`Reflections`) seems more appropriate because it refers to the library/class.